src/sentry/static/sentry/app/components/charts/eventsRequest.tsx

```diff
@@ -202,18 +210,29 @@ class EventsRequest extends React.PureComponent<EventsRequestProps, EventsReques
       errored: false,
     }));
 
-    try {
-      api.clear();
-      timeseriesData = await doEventsRequest(api, props);
-    } catch (resp) {
-      if (resp && resp.responseJSON && resp.responseJSON.detail) {
-        addErrorMessage(resp.responseJSON.detail);
-      } else {
-        addErrorMessage(t('Error loading chart data'));
-      }
+    if (expired) {
+      addErrorMessage(
+        t(name + ' has an invalid date range. Please try a more recent date range.'),
```

Comment:
```suggestion
        t('%s has an invalid date range. Please try a more recent date range.', name),
```

The translation message extractor needs placeholders. Concatenating values together doesn't work as nouns can move around based on the locale.